src/components/AboutComp.jsx

```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { Outlet } from "react-router-dom";
+
+class AboutComp extends React.Component {
+  constructor(props) {
+    super(props);
+    console.log("About -Constructor");
+  }
+  componentDidMount() {
+    console.log("About - componentDidMount");
+  }
+  render() {
+    console.log("About Render");
+    return (
+      <div>
+        AboutComp
```

Comment:
angular brackets missing here (AboutComp) or is it supposed to be like this only?